scst-store/ingress-multicluster.hbs.md

```diff
@@ -56,6 +61,32 @@ Set up the cluster containing Supply Chain Security Tools - Store first and enab
 
 To get Supply Chain Security Tools - Store's TLS CA certificate, run:
 
+```bash
```

Comment:
is this the case for self signed cert?

Reply:
I put <cert-name> in the command, so that they just put in the name depending on whether they are using self signed or custom. Do you think that's kind of confusing? I can just separate it into two cases if it is

Reply:
i think this is fine